src/Models/HydrostaticFreeSurfaceModels/z_star_vertical_spacing.jl

```diff
@@ -12,55 +13,117 @@ barotropic_velocities(free_surface::SplitExplicitFreeSurface) = free_surface.bar
 barotropic_velocities(free_surface) = nothing, nothing
 
 # Fallback
-update_grid!(model, grid, ztype; parameters) = nothing
+ab2_step_grid!(grid, model, ztype, Δt, χ) = nothing
 
-function update_grid!(model, grid::MutableGridOfSomeKind, ::ZStar; parameters = :xy)
+function ab2_step_grid!(grid::MutableGridOfSomeKind, model, ::ZStar, Δt, χ)
```

Comment:
model knows the time stepper so why `ab2_step_grid!` and  not just `step_grid!` and dispatch over the time stepper?

Reply:
Perhaps doing so it's cleaner? Not sure. I'm happy either way. Wanted to see if there was some other reason I couldn't see myself.